### PR TITLE
debug:  add all packages required for the collect-info to the Dockerfile

### PR DESCRIPTION
**Pull request by rouming**
Now debug container has all the packages, so on latest EVE script should not depend on network. But script sill tries to install all packages in run-time to keep old versions of EVE working fine.

### CODE REVIEW

File: pkg/debug/scripts/collect-info.sh [hunk 1]
```diff
@@ -5,10 +5,10 @@
 #
 
 # Script version, don't forget to bump up once something is changed
-VERSION=6
+VERSION=7
 
 # Add required packages here, it will be passed to "apk add".
-PKG_DEPS="procps dmidecode iptables dhcpcd"
+PKG_DEPS="procps tar dmidecode iptables dhcpcd"
```
Comment (rene) [inline pkg/debug/scripts/collect-info.sh:16]: In your commit message: "This is correct **which** one exception" -> "This is correct **with** one exception

**Review by rene — COMMENTED** [review]
LGTM

**Review by eriknordmark — APPROVED** [review]
LGTM apart from René's typo

File: pkg/debug/scripts/collect-info.sh (hunk 1 above)
Comment (rouming) [reply pkg/debug/scripts/collect-info.sh:16]: Yep, thanks. Will fix

File: pkg/alpine/mirrors/3.16/main [hunk 2]
```diff
@@ -134,22 +136,29 @@ mtools
...
 p11-kit
+parted
 patch
 pciutils
 perl
 pixman-dev
 pkgconf
+pm-utils
 ppp
+procps
```
Comment (eriknordmark) [inline pkg/alpine/mirrors/3.16/main:161]: @rouming Turns out this package doesn't exist for riscv so the build fails. Maybe I missed that for the PR build or I was confused by the arm64 failure due to no runner?

File: pkg/debug/Dockerfile [hunk 3]
```diff
@@ -4,14 +4,14 @@
...
-FROM lfedge/eve-alpine:9fb9b9cbf7d90066a70e4704d04a6fe248ff52bb as build
+FROM lfedge/eve-alpine:fad44e3702708a8d044663a20fd98d933dddb41e as build
 ENV BUILD_PKGS abuild curl tar make linux-headers patch g++ git gcc ncurses-dev autoconf
 # Feel free to add additional packages here, but be aware that
 # EVE's rootfs image can be no larger than 300Mb (and don't
 # forget to check on all supported architectures: e.g. arm64
 # binaries are typically larger and amd64 ones).
 # RUN apk add --no-cache gdb valgrind
-ENV PKGS openssl openssh-client openssh-server tini util-linux ca-certificates pciutils usbutils vim tcpdump perf strace iproute2-minimal curl tar
+ENV PKGS openssl openssh-client openssh-server tini util-linux ca-certificates pciutils usbutils vim tcpdump perf strace iproute2-minimal curl procps tar dmidecode iptables dhcpcd
```
Comment (eriknordmark) [inline pkg/debug/Dockerfile:14]: This might fail on riscv as well due to procps not existing

**Pull request merged**